conference/react-js/src/containers/Conference-cards/Past/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import axios from 'axios';
+import baseUrl from '../../../constants/backend-url';
+import Card from '../../../components/Card';
+
+class Past extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: null,
+    };
+  }
+
+  componentWillMount() {
+    axios.get(`${baseUrl}/api/conference/Past`)
```

Comment:
Make API calls in componentDidMount 
https://daveceddia.com/where-fetch-data-componentwillmount-vs-componentdidmount/